src/Controller/QuestionController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\QuestionManager;
+
+class QuestionController extends AbstractController
+{
+
+    /**
+     * @return string
+     * Displays a random question and its possible choices.
+     */
+    public function index(): string
+    {
+        $questionManager = new QuestionManager();
+        $question = $questionManager->selectOneRandom();
```

Comment:
Wouldn't it be more convenient to get the question and its answers using only one query ? (a JOIN)

If you stick with this strategy, you should check if you really get a question before querying for the related answers.